sdcm/nemesis.py

```diff
@@ -2775,11 +2776,12 @@ def disrupt_remove_node_then_add_node(self):  # pylint: disable=too-many-branche
         assert removed_node_status is not None, "failed to get host_id using nodetool status"
         host_id = removed_node_status["host_id"]
 
-        # node stop and make sure its "DN"
-        node_to_remove.stop_scylla_server(verify_up=True, verify_down=True)
+        with ignore_ycsb_connection_refused():
+            # node stop and make sure its "DN"
+            node_to_remove.stop_scylla_server(verify_up=True, verify_down=True)
```

Comment:
@fruch , why adding this to nemesis.py? why not adding to the method `stop_scylla_server` itself in cluster.py?

Reply:
since in this we need to cover more then the stop itself, also the termination.

anyhow it's mostly to be align with all of the other places we add such things.

also doing that in `stop_scylla_server` we won't have any method to disable it in specific cases, if needed.

Reply:
at a some point we should move to the client base load balancing for alternator, and those should be removed.